configure.ac

```diff
@@ -302,6 +302,17 @@ AC_ARG_WITH([cc-image],
     ]
 )
 
+AC_ARG_ENABLE([autogopath],
+	AS_HELP_STRING([--enable-autogopath], [enable autogopath @<:@default=false@:>@]),
```

Comment:
Sure - I know what it's for, but I think it would be useful to specify those words inside the `AS_HELP_STRING()` call as folk outside the project who want to build the code might wonder why we have this option ;)